### PR TITLE
feat: add saber-plugin-pwa

### PR DESCRIPTION
**Pull request by egoist**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI/CSS related code, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: website/pages/docs/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Introduction
 layout: docs
 ---
 
-Saber is a static website generator, it is like Hexo but, you can use Vue, React or other view layers[^1] to write layouts, Saber will also generate a universal website[^2] out-of-the-box.
+Sabe!!x is a static website generator, it is like Hexo but, you can use Vue, React or other view layers[^1] to write layouts, Saber will also generate a universal website[^2] out-of-the-box.
```
Comment (h404bi) [inline website/pages/docs/index.md]: typo?

Comment (egoist) [reply website/pages/docs/index.md]: just testing the update notifier

**Comment by egoist** [comment]
Published canary version: saber-plugin-pwa@0.1.1-canary.392+8a4c6c9

**Comment by egoist** [comment]
Published canary version: saber-plugin-pwa@0.1.1-canary.400

**Pull request merged**